includes/base_controls/QControlProxy.class.php

```diff
@@ -34,7 +32,7 @@ public function RenderAsEvents($strActionParameter = null, $blnDisplayOutput = t
 			else
 				$this->strTargetControlId = $this->objForm->GenerateControlId();
 				
-			$this->strActionParameter = $strActionParameter;
+			$this->mixActionParameter = $strActionParameter;
```

Comment:
mixActionParameter is not defined in the class. Please define it. Not doing so produces a soft warning in PhpStorm which can be confusing at best and scaring in worse cases. Please add it the way strActionParameter was present.